coldfront/plugins/qumulo/tests/test_tasks.py

```diff
@@ -245,3 +261,256 @@ def test_conditionally_update_storage_allocation_statuses_checks_all_pending_all
+        for path, quoata_data in base_quotas.items():
+            storage_filesystem_path_attribute = AllocationAttribute.objects.select_related(
+                "allocation"
+            ).get(
+                value=path.rstrip("/"),
+                allocation_attribute_type=self.storage_filesystem_path_attribute_type,
+                allocation__status=self.status_active,
+            )
+
+            allocation = storage_filesystem_path_attribute.allocation
+            storage_quota_attribute = AllocationAttribute.objects.get(
+                allocation=allocation,
+                allocation_attribute_type=self.storage_quota_attribute_type,
+            )
+
+            allocation_attribute_usage = (
+                storage_quota_attribute.allocationattributeusage
+            )
+
+            usage = int(quoata_data["usage"])
+            self.assertEqual(allocation_attribute_usage.value, usage)
+            self.assertEqual(allocation_attribute_usage.history.first().value, usage)
+            self.assertGreater(allocation_attribute_usage.history.count(), 1)
+
+    @patch("coldfront.plugins.qumulo.tasks.QumuloAPI")
+    def test_doesnt_ingest_sub_allocation_data(
+        self, qumulo_api_mock: MagicMock
+    ) -> None:
+        qumulo_api = MagicMock()
+        qumulo_api.get_all_quotas_with_usage.return_value = self.mock_quota_response
+        qumulo_api_mock.return_value = qumulo_api
+
+        try:
+            ingest_quotas_with_daily_usage()
+        except:
+            self.fail("Ingest quotas raised exception")
+
+        sub_quotas = get_mock_quota_sub_allocations(self.STORAGE2_PATH)
+        for path in sub_quotas.keys():
+            allocation_attribute_usage = None
+            storage_filesystem_path_attribute = AllocationAttribute.objects.get(
+                value=path.rstrip("/"),
+                allocation_attribute_type=self.storage_filesystem_path_attribute_type,
+            )
+
+            allocation = storage_filesystem_path_attribute.allocation
+            storage_quota_attribute = AllocationAttribute.objects.get(
+                allocation=allocation,
+                allocation_attribute_type=self.storage_quota_attribute_type,
+            )
+
+            allocation_attribute_usage = (
+                storage_quota_attribute.allocationattributeusage
+            )
+
+            self.assertEqual(allocation_attribute_usage.value, 0)
+            self.assertEqual(allocation_attribute_usage.history.first().value, 0)
+
+    @patch("coldfront.plugins.qumulo.tasks.QumuloAPI")
+    def test_filtering_out_not_active_allocations(
+        self, qumulo_api_mock: MagicMock
+    ) -> None:
+        index = 1
+        path = f"{self.STORAGE2_PATH}/status_test"
+        limit = "100000000000000"
+        capacity_usage = "37089837494272"
+
+        mock_quota = {
+            "quotas": [
+                {
+                    "id": index,
+                    "path": path,
```

Comment:
@harterjwustl 
 This was my doing. The path in QUMULO always has a trailing slash.
I wanted to go on the record: I would let it be.

Reply:
can you be more clear on what you would let be?

I structured the mock data so that the path's always *have* a trailing slash and I stripped them when mocking allocation data in coldfront (vs not stripping them here for qumulo data)

Reply:
Leave it as it is. The QUMULO path should have a trailing slash, but it is not a deal breaker.